tools/smaps-view.py

```diff
@@ -0,0 +1,194 @@
+"""
+Usage examples:
+  # Basic usage - show aggregated cache percentages for all vector index files
+  python smaps-view.py $(pidof qdrant) '.*/vector_index/.*'
+
+  # Verbose mode - show individual files larger than 1MB
+  python smaps-view.py $(pidof qdrant) '.*/vector_storage/.*' -v
+
```

Comment:
```suggestion
  # Verbose mode - show vector storage files larger than 1MB
  python smaps-view.py $(pidof qdrant) '.*/vector_storage/.*' -v

  # Verbose mode - show all files larger than 1MB
  python smaps-view.py $(pidof qdrant) '.*' -v
```

I tested on a small collection. Needed `python smaps-view.py $(pidof qdrant) '.*' -v` to see file stats. Might be better to highlight that in docs.

Reply:
what is the point? This "match_anything" pattern will include a lot of garbage